### ISSUE TITLE
Airbrake reporting ALL urls as secure with a port

### ISSUE THREAD

**Issue by noctivityinc** [airbrake/airbrake#802]
We upgraded from Rails 3.2 to Rails 5.1.4 and now all URLs are being reported as secure but with port 80, for example `https://www.dmvcheatsheets.com:80/`

We're using the latest airbrake and airbrake-ruby gems.

It's happening for EVERY error report, so I dont really have steps to reproduce.  Here is my airbrake.rb initializer.

```
Airbrake.configure do |config|
  config.project_key = 'XXX...'
  config.project_id = 83646
  config.root_directory = Rails.root
  config.environment = Rails.env
  config.ignore_environments = %w(development test)
end

Airbrake.add_filter do |notice|
  notice[:environment].merge!(ENV) # sends env information
  if notice[:errors].any? { |error| error[:type] == 'MethodNotAllowed' || error[:type].include?('IpSpoofAttackError') }
    notice.ignore!
  end
end
```

**Comment by kyrylo** [comment]
Do you use Thin?

**Comment by noctivityinc** [comment]
We use Unicorn with nginx.   
  

  
  

  
  
>

**Comment by kyrylo** [comment]
Thanks. This should be fixed, however I haven't released a new version yet.

**Comment by noctivityinc** [comment]
Ok I’ll look out for it.   
  

  
  

  
  
>

**Comment by kyrylo** [comment]
@noctivityinc I just released v7.0.3 with this fix. Enjoy!
